tests/functional/tnf/test_tnf_deployment.py

```
@@ -0,0 +1,570 @@
+            time.sleep(10)
+
+        assert (
+            mon_recreated
+        ), f"Monitor pod not recreated on {mon_node} within {max_wait}s"
+
+        logger.test_step("Verify all monitors rejoin quorum")
+        time.sleep(30)
+        quorum_final = json.loads(
+            ceph_cluster.toolbox.exec_cmd_on_pod("ceph quorum_status --format json")
+        )
+        logger.info(f"Final quorum members: {quorum_final.get('quorum_names', [])}")
```

Ruff (0.16.2)

[warning] Logging statement uses f-string

(G004)